resources/markdown/manual.md

```diff
@@ -40,16 +46,16 @@ This document provides guidelines how to use ProBro extension.
 
 ## 4. Fields and Indexes explorer
 
-  - Select table to see fields and indexes. Fields and Indexes are displayed in panel view. By default, they are on separate tabs, but can be merged by dragging.
+-   Select table to see fields and indexes. Fields and Indexes are displayed in panel view. By default, they are on separate tabs, but can be merged by dragging.
 
 <img src="../images/mergeFieldsIndexes.gif" height="150">
 
- - In fields explorer has multi-sorting and multi-filtering options.
-Multi-sorting can be used by holding CTRL key and selecting preferred columns. 
+-   In fields explorer has multi-sorting and multi-filtering options.
+    Multi-sorting can be used by holding CTRL key and selecting preferred columns.
 
 <img src="../images/multisorting.gif">
 
-- By selecting field rows, query columns can be shown/hidden.
+-   By selecting field rows, query columns can be shown/hidden.

```

Comment:
Mention that this selection is save for a user. Also, that RECID, ROWID records fields are by default hidden, but can be enabled here.